pages/home/index.tsx

```diff
@@ -27,25 +27,32 @@ export default function Home() {
     setTasks(tasks ? tasks : []);
   }, []);
 
-  useEffect(() => {
-    addEventListener("keypress", ({ key }: any) => {
-      if (key == "Enter") {
-        handleClick();
-      }
-    });
-  }, []);
+
 
   const handleClick = () => {
     let tasks: any = JSON.parse(
       // @ts-ignore
       localStorage.getItem("tasks") ? localStorage.getItem("tasks") : []
     );
```

Comment:
the exception should be here

```suggestion
try{
    let tasks: any = JSON.parse(
      // @ts-ignore
      localStorage.getItem("tasks") ? localStorage.getItem("tasks") : []
    );
   } catch (err) {
     let tasks: any = [];
     setTasks([]);
   }
```